easybuild/easyconfigs/k/Kent_tools/Kent_tools-479-GCC-13.3.0.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'MakeCp'
+
+name = 'Kent_tools'
+version = '479'
+
+homepage = 'https://genome.cse.ucsc.edu/'
+description = """Kent utilities: collection of tools used by the UCSC genome browser."""
+
+toolchain = {'name': 'GCC', 'version': '13.3.0'}
+source_urls = [
+    'https://hgdownload.cse.ucsc.edu/admin/exe/',
+    'https://hgdownload.cse.ucsc.edu/admin/exe/userApps.archive/',
+]
+sources = ['userApps.v%(version)s.src.tgz']
```

Comment:
Checksum is missing here, but for some reasons that's not caught by easyconfigs test suite?!